### PR TITLE
Cookd Webhooks

### PR DESCRIPTION
**Pull request by coreyja**

### CODE REVIEW

File: server/src/http_server/webhooks/cookd.rs [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+    let db_result = sqlx::query!(
+        r#"
+        INSERT INTO CookdWebhooks (subdomain, slug, player_github_email, player_github_username, score)
+        VALUES ($1, $2, $3, $4, $5) RETURNING cookd_webhook_id
+        "#,
+        payload.subdomain,
+        payload.slug,
+        payload.player_github_email,
+        payload.player_github_username,
+        payload.score,
```
Comment (aalobai1) [inline server/src/http_server/webhooks/cookd.rs:36]: I'm currently using camel case for the webhook payload but can chance it to snake_case to save you some lines on transforming it. It seems like most people would prefer snake_case anyways?

**Review by aalobai1 — COMMENTED** [review]
this PR reminds me of ruby backends and how much I miss them ... lowkey want to write a backend in rust now for fun 😆

File: server/src/http_server/webhooks/cookd.rs (hunk 1 above)
Comment (coreyja) [reply server/src/http_server/webhooks/cookd.rs:36]: 🤷  Not a big deal to me lol

You'll see in my next push that its just one line to tell it to serialize/deserilize to camelCase and I can keep my Rusty snake case variables 🎉

File: server/src/http_server/webhooks/cookd.rs [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+use cja::{app_state::AppState as _, color_eyre::eyre::Context};
+use serde_json::{json, Value};
+
+use crate::{
+    http_server::{errors::WithStatus as _, ResponseResult},
+    AppState,
+};
+
+#[derive(serde::Deserialize, Clone)]
+#[serde(rename_all = "camelCase")]
```
Comment (coreyja) [inline server/src/http_server/webhooks/cookd.rs:11]: @aalobai1 and now I can handle the camelCase 🎉  lol

**Comment by coreyja** [comment]
> this PR reminds me of ruby backends and how much I miss them ... lowkey want to write a backend in rust now for fun 😆

DOO ITTT 😉  lol

**Comment by coreyja** [comment]
@aalobai1 Tried this out and don't think I got the webhook on my side? Can you maybe check your logs and see if you made the request?

Set the URL for this cookd: https://coreyja.cookd.dev/level-0-0
The webhook url is set to: https://coreyja-com-pr-71.fly.dev/webhooks/cookd

Don't see a result in my table, or any errors/requests in my logs (but not sure I have the best logging today lol)

**Comment by coreyja** [comment]
Ok so I _did_ find my logs, just missed them the first time and my search must suck lol

It was definitely a me issue 😆  Will confirm its fixed when it deploys in a sec, but I didn't have a DB default on my id

**Comment by coreyja** [comment]
Took me a bit cause I kept forgetting that the webhook setting was per question so I made a few questions and forgot to set it before answering them 😆 

But got a DB record finally to confirm this is working!

**Review by Seif-Mamdouh — COMMENTED** [review]
looks good! I'm excited!

**Pull request merged**